SharedTests/AuthenticationKeychainInfoTests.swift

```diff
@@ -0,0 +1,45 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Foundation
+import Shared
+import XCTest
+import SwiftKeychainWrapper
+
+class AuthenticationKeychainInfoTests: XCTestCase {
+
+    func testEncodingAndDecoding() {
+        let passcode = "1234"
```

Comment:
This test. Tests all the params. Making sure they are encoded/decoded properly 👍